context_prop/src/main/java/io/opentelemetry/context/propagation/DefaultContextPropagators.java

```diff
@@ -123,12 +127,12 @@ private MultiTextMapPropagator(List<TextMapPropagator> textPropagators) {
     }
 
     private static List<String> getAllFields(TextMapPropagator[] textPropagators) {
-      List<String> fields = new ArrayList<>();
+      Set<String> fields = new HashSet<>();
```

Comment:
Let's use `LinkedHashSet` to preserve order, users may have a reason to iterate in a specified order.

Reply:
done